### PR TITLE
[AnomalyDetector] Generate Python SDK from CADL spec

### PR DESCRIPTION
**Pull request by catalinaperalta**
Generated based off of the following CADL spec: https://github.com/Azure/azure-rest-api-specs/pull/21525
Tracking issues: #27513 
ApiView: https://apiview.dev/Assemblies/Review/a97ff6898fb443ee82a679e0ac1aedd6?diffRevisionId=cee43f5f3d3345c594c526675a225853&diffOnly=False&revisionId=5f7debea09204d8097f90998ff86a520&doc=False

### CODE REVIEW

File: sdk/anomalydetector/azure-ai-anomalydetector/azure/ai/anomalydetector/_client.py [hunk 1]
```diff
@@ -17,42 +16,46 @@
...
     :param credential: Credential needed for the client to connect to Azure. Required.
     :type credential: ~azure.core.credentials.AzureKeyCredential
     :keyword api_version: Api Version. Default value is "v1.1". Note that overriding this default
      value may result in unsupported behavior.
     :paramtype api_version: str
     """
 
-    def __init__(self, endpoint: str, credential: AzureKeyCredential, **kwargs: Any) -> None:
+    def __init__(
+        self, endpoint: str, api_version: Union[str, _models.APIVersion], credential: AzureKeyCredential, **kwargs: Any
```
[the remark this answers is not in the document]
Comment (catalinaperalta) [reply sdk/anomalydetector/azure-ai-anomalydetector/azure/ai/anomalydetector/_client.py:53]: There's another issue with endpoints that I reported to Izzy. I'll wait on updating until those fixes are out.

[the remark this answers is not in the document]
Comment (catalinaperalta) [reply sdk/anomalydetector/azure-ai-anomalydetector/azure/ai/anomalydetector/_client.py:53]: Tracking issue is in the description, will resolve this.

File: sdk/anomalydetector/azure-ai-anomalydetector/azure/ai/anomalydetector/models/_models.py [hunk 2]
```diff
@@ -0,0 +1,1409 @@
...
+from typing import Any, List, Mapping, Optional, TYPE_CHECKING, Union, overload
+
+from .. import _model_base
+from .._model_base import rest_field
+
+if sys.version_info >= (3, 9):
+    from collections.abc import MutableMapping
+else:
+    from typing import MutableMapping  # type: ignore  # pylint: disable=ungrouped-imports
+
```
Comment (msyyc) [inline sdk/anomalydetector/azure-ai-anomalydetector/azure/ai/anomalydetector/models/_models.py:21]: Could you add some test? Here is the guidance: https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-for-Test

Comment (catalinaperalta) [reply sdk/anomalydetector/azure-ai-anomalydetector/azure/ai/anomalydetector/models/_models.py:21]: @jr-MS and team will take over this branch and add tests and samples.

Comment (msyyc) [reply sdk/anomalydetector/azure-ai-anomalydetector/azure/ai/anomalydetector/models/_models.py:21]: @jr-MS Could you help add test for `model`?

**Comment by msyyc** [comment]
Hi @Xingyu-Yue Could you help update SDK code with latest cadl after https://github.com/Azure/azure-rest-api-specs/pull/21525 merged

**Comment by msyyc** [comment]
Hi @annatisch The SDK code has been updated after [ArcBoard review](https://apiview.dev/Assemblies/Review/a97ff6898fb443ee82a679e0ac1aedd6?diffRevisionId=cee43f5f3d3345c594c526675a225853&diffOnly=False&revisionId=5f7debea09204d8097f90998ff86a520&doc=False). Could you help review the PR? They want to release it in these days:
image

**Review by annatisch — APPROVED** [review]

**Pull request merged**